### PR TITLE
[CFP-276] Investigate Fixes/Alternatives for affected libraries 

### PR DESCRIPTION
**Pull request by imtiazAhmd**
#### What
After the jQuery upgrade to 3.6.0, Some of our third parties libraries are found to be using deprecated methods. We need to find fixes for these libraries or find alternatives.
#### Ticket

[Investigate suitable alternatives for affected Third-Party Libraries due to the jQuery upgrade  (1/2)](https://dsdmoj.atlassian.net/browse/CFP-276)

#### TODO (wip)

Fixes for third party libraries
 - [x] cocoon.js
 - [x] jquery.numbered.elements.js
 - [x] typeahead-aria.js
---------- 
Fixes for `stick-at-top-while-scrolling`
- [x] CSS only implementation
- [x] include Polyfill to support IE11
----------
Cleanup
- [x] Use `<aside>` HTML5 element
- [x] Fix Failing Tests
- [x] Remove jqueryMigrate
- [x] Sanity check in Dev Env
- [x] Sanity check using Browser Stack to check IE11

### CODE REVIEW

File: app/webpack/javascripts/vendor/cocoon.js [hunk 1]
```diff
@@ -132,13 +132,17 @@
   });
 
 
-  $(document).on("ready page:load turbolinks:load", function() {
-    $('.remove_fields.existing.destroyed').each(function(i, obj) {
-      var $this = $(this),
-          wrapper_class = $this.data('wrapper-class') || 'nested-fields';
+var hideRemoveFields = function (params) {
```
Comment (naseberry) [inline app/webpack/javascripts/vendor/cocoon.js:135]: As discussed, I would suggest a better variable name

File: app/webpack/javascripts/vendor/typeahead-aria.js [hunk 2]
```diff
@@ -2463,7 +2471,7 @@
             });
         }
         function buildHintFromInput($input, www) {
-            return $input.clone().addClass(www.classes.hint).removeData().css(www.css.hint).css(getBackgroundStyles($input)).prop("readonly", true).removeAttr("id name placeholder required").attr({
+            return $input.clone().addClass(www.classes.hint).removeData().css(www.css.hint).css(getBackgroundStyles($input)).prop("readonly", true).removeAttr('id').removeAttr('name').removeAttr('placeholder').removeAttr('required').attr({
```
Comment (naseberry) [inline app/webpack/javascripts/vendor/typeahead-aria.js]: As discussed, you may wish to undo this.

File: app/webpack/stylesheets/_layout.scss [hunk 3]
```diff
@@ -72,3 +72,16 @@ body {
   @include govuk-responsive-margin(9, "bottom");
   position: relative;
 }
+
+// Replacement for stick-at-top-when-scrolling
```
Comment (naseberry) [inline app/webpack/stylesheets/_layout.scss:76]: 💯  
No need for JS

File: app/webpack/javascripts/vendor/typeahead-aria.js (hunk 2 above)
Comment (imtiazAhmd) [reply app/webpack/javascripts/vendor/typeahead-aria.js]: done

File: app/webpack/javascripts/vendor/cocoon.js (hunk 1 above)
Comment (imtiazAhmd) [reply app/webpack/javascripts/vendor/cocoon.js:135]: we are hiding `.remove_fields` with this function. Any suggestion for the variable name?

**Review by naseberry — APPROVED** [review]
Looks good.
I've sanity checked changes locally and on a hosted env.
Sticky tested on IE11.
Views affected by individual file changes sanity checked on hosted env.
💯

**Pull request merged**